packages/core/parcel-bundler/test/javascript.js

```diff
@@ -115,6 +116,12 @@ describe('javascript', function() {
     assert(
       JSON.parse(pkg).devDependencies['@babel/plugin-proposal-class-properties']
     );
+
+    // cleanup node_modules, otherwise the next time we run the tests
+    // it doesn't add it to the "package.json" cause it exists in the node_modules when required
+    await rimraf(
```

Comment:
Not sure but perhaps we should move the babel autoinstall test to the input folder before running it like we do with other tests that create files